src/main/scala/temple/builder/KubernetesBuilder.scala

```diff
@@ -0,0 +1,38 @@
+package temple.builder
+
+import temple.DSL.semantics.Metadata.Database
+import temple.DSL.semantics.{Metadata, ServiceBlock}
```

Comment:
```suggestion
import temple.ast.Metadata.Database
import temple.ast.{Metadata, ServiceBlock}
```
The merge 6e84646 broke this